.htaccess

```diff
@@ -29,9 +29,6 @@
   Header append Vary User-Agent
 </IfModule>
 
-AddType text/cache-manifest .manifest
-AddType text/cache-manifest .appcache
```

Comment:
We should drop this entire file. it doesn't get used

Reply:
True, I suppose it's leftover from a time this was once served by Apache. Deleted.